readthedocs/doc_builder/python_environments.py

```diff
@@ -236,7 +236,7 @@ def install_core_requirements(self):
             'recommonmark==0.4.0',
         ]
 
-        if self.project.documentation_type == 'mkdocs':
+        if self.config.doctype == 'mkdocs':
```

Comment:
I'm leaving the doctype access in some places where we always use the config object, it doesn't break anything anyway. Just keeping the same pattern.